<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VSPowerMenu</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="sleepButton">
      <property name="text">
       <string>  Sleep</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="logoutButton">
      <property name="text">
       <string>  Logout</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rebootButton">
      <property name="text">
       <string>  Reboot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="shutdownButton">
      <property name="text">
       <string>  Shutdown</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
